scalafmt-tests/src/test/resources/scala3/OptionalBraces_keep.stat

```diff
@@ -3424,3 +3424,66 @@ object a:
       val x = 4
       4 == 3
    do 5
+<<< #3276
+class Foo() {
+  def ok: Boolean =
+    if (1 == 1) {
+      true
+    } else
+      false
+
+  def notOK: Boolean =
+    if (1 == 1) {
+      println("Some code")
+      true
+    } else
+      false
+}
+>>>
+class Foo() {
+  def ok: Boolean =
+    if (1 == 1) {
+      true
+    } else
+      false
+
+  def notOK: Boolean =
+    if (1 == 1) {
+      println("Some code")
+      true
+    } else
+      false
+}
+<<< #3276 2
+class Foo() {
+  def notOK: Boolean =
+    if (1 == 1)
+      {c; a + b}.foo
+    else
+      false
+}
+>>>
+class Foo() {
+  def notOK: Boolean =
+    if (1 == 1) { c; a + b }.foo
```

Comment:
Shouldn't we keep the newline here ?

Reply:
same comment about an old rule as above.